.github/workflows/build.yml

```
@@ -0,0 +1,50 @@
+name: MicroProfile Telemetry Build
+
+on:
+  push:
+    paths-ignore:
+      - 'editorconfig'
+      - '.gitattributes'
+      - '.gitignore'
+      - 'CONTRIBUTING*'
+      - 'CODEOWNERS'
+      - 'KEYS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+      - 'site.yaml'
+  pull_request:
+    paths-ignore:
+      - 'editorconfig'
+      - '.gitattributes'
+      - '.gitignore'
+      - 'CONTRIBUTING*'
+      - 'CODEOWNERS'
+      - 'KEYS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+      - 'site.yaml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    name: build with jdk ${{matrix.java}}
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+        name: checkout
+        with:
+          ref: ${{ github.event.pull_request.merge_commit_sha }}
+
+      - uses: actions/setup-java@v2
+        name: set up jdk ${{matrix.java}}
+        with:
+          distribution: adopt
```

Should we move to the new one?
```suggestion
          distribution: temurin
          java-version: 8
```

---

The java-version was taken from the matrix [8, 11]